.github/workflows/release.yml

```diff
@@ -0,0 +1,159 @@
+    name: Release Linux
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Cache dependencies
+        id: cache-dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Get dependencies
+        if: steps.cache-dependencies.outputs.cache-hit != 'true'
+        run: |
+          export PATH=${PATH}:`go env GOPATH`/bin
+          go get -v -t -d ./...
+      - name: Install gox
+        run: |
+             export PATH=${PATH}:`go env GOPATH`/bin
+             go get github.com/mitchellh/gox  
+      - name: Compile
+        run: |
+              export PATH=${PATH}:`go env GOPATH`/bin
+              SUMMARY=${GITHUB_SHA}
+              if [[ $GITHUB_REF == refs/tags/* ]]; then
+                SUMMARY=${GITHUB_REF/refs\/tags\//}
+              fi  
+              DATE=$(date --iso-8601=seconds)
+              FLAGS="-X github.com/textileio/go-threads/common.GitSummary=${SUMMARY} -X github.com/textileio/go-textile/common.BuildDate=${DATE} -X github.com/textileio/go-textile/common.GitCommit=${GITHUB_SHA} -X github.com/textileio/go-textile/common.GitBranch=master -X github.com/textileio/go-textile/common.GitState=clean"
+              gox -ldflags="-w $FLAGS" -osarch="linux/amd64 linux/386 linux/arm" -output="threadsd-{{.OS}}-{{.Arch}}" ./threadsd
+              gox -ldflags="-w $FLAGS" -osarch="linux/amd64 linux/386 linux/arm" -output="threads-{{.OS}}-{{.Arch}}" ./threads
+      - name: Collect artifacts
+        run: |
+          VERSION=${GITHUB_REF##*/}
+          OUT=~/dist/cli
+          mkdir -p ${OUT}
+          mkdir -p tmp
+          cp LICENSE tmp/
+          cp dist/README.md tmp/
+          cp dist/install tmp/
+          cd tmp
+          declare -a arr=("linux-amd64" "linux-386" "linux-arm")
+          for i in "${arr[@]}"
+          do
+              OSARCH=${i%.*}
+              cp ../threadsd-${i} threadsd
+              cp ../threads-${i} threads
+              tar -czvf go-threads_${VERSION}_${OSARCH}.tar.gz LICENSE README.md install threadsd${EXT}
+              mv go-threads_${VERSION}_${OSARCH}.tar.gz ${OUT}/
+          done
+          cd .. && rm -rf tmp
+      - name: Mock Publish
+        run: |
+              OUT=~/dist/cli
+              echo $(ls ${OUT}/)
+      # - name: Upload multiple assets to release
+      #   uses: radiusnetworks/release-asset-action@v1
+      #   with:
+      #     files: |
+      #       dist/cli/go-threads_${GITHUB_REF##*/}_linux-amd64.tar.gz
+      #       dist/cli/go-threads_${GITHUB_REF##*/}_linux-386.tar.gz
+      #       dist/cli/go-threads_${GITHUB_REF##*/}_linux-arm.tar.gz
+      #     github-token: ${{ secrets.GITHUB_TOKEN }}
+  release-win-mac:
```

Comment:
Since we don't need `CGO` enabled, you don't need to supply a C toolchain. In other words, you should be able to just build the mac and windows versions on the linux machine above with `gox`.

Reply:
okay, fixed it up. looks like these machines are CPU=1 right now